project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.module

```diff
@@ -314,6 +314,11 @@ function c4m_user_profile_form_user_register_form_alter(
     ),
   );
   $form['#submit'][] = 'c4m_user_profile_form_user_register_form_submit';
+
+  // Hide status fields - SA-created user is always active.
```

Comment:
@ordavidil 
Status field does not appear on form, for non admin users.
The options are 'Active' and 'Blocked', and user may not choose it's status.

![selection_008](https://cloud.githubusercontent.com/assets/17882245/17143317/b0e70d1e-535b-11e6-976e-c7641ecaa9fe.png)